crates/bevy_pbr/src/wireframe.rs

```diff
@@ -81,19 +101,56 @@ struct GlobalWireframeMaterial {
 fn setup_global_wireframe_material(
     mut commands: Commands,
     mut materials: ResMut<Assets<WireframeMaterial>>,
+    config: Res<WireframeConfig>,
 ) {
     // Create the handle used for the global material
     commands.insert_resource(GlobalWireframeMaterial {
-        handle: materials.add(WireframeMaterial {}),
+        handle: materials.add(WireframeMaterial {
+            color: config.default_color,
+        }),
     });
 }
 
+/// Updates the wireframe material of all entities without a [`WireframeColor`] or without a [`Wireframe`] component
+fn global_color_changed(
+    config: Res<WireframeConfig>,
+    mut materials: ResMut<Assets<WireframeMaterial>>,
+    global_material: Res<GlobalWireframeMaterial>,
+) {
+    if !config.is_changed() {
```

Comment:
https://dev-docs.bevyengine.org/bevy/ecs/prelude/fn.resource_changed.html

Reply:
Good choice: run conditions will be marginally more performant (and better reflect intent).